src/Nethermind/Nethermind.Db/DbOnTheRocks.cs

```diff
@@ -222,6 +223,22 @@ public void Remove(byte[] key)
             _db.Remove(key);
         }
 
+        public byte[][] GetAll()
```

Comment:
this should not be in the base class - can be extending - something like IterableDbOnTheRocks : IIterableDb